public/js/components/Rep.js

```diff
@@ -0,0 +1,13 @@
+"use strict";
+
+
+// Node does not support the AMD syntax used in Rep
+// Node also does not support Worker, which is an issue when I built the Rep with webpack
+if (typeof window == "object") {
+    const Rep = require("../lib/devtools/client/shared/components/reps/Rep").Rep;
+    require("../lib/devtools/client/shared/components/reps/reps.css");
+
+    module.exports = Rep;
+} else {
+    module.exports = function() {};
+}
```

Comment:
The unit tests fail in Node for these two reasons. @janodvarko do you have recommendations on what we can do? It seems like the easiest thing to do is to add a typeof check for Worker and then build Rep. The ideal solution would be to convert Rep to commonjs, but I'm not sure if that would be possible...